test/packages/benchmarks/system_benchmark/_dev/benchmark/system/20mb-logs-benchmark/config.yml

```diff
@@ -0,0 +1,40 @@
+- name: IP
```

Comment:
Is it by design that we don't use here names "similar" to ECS like `source.ip`? I'm aware this is Schema A so not ECS yet at the same time using ECS could make it easy to read (if possible). 

This is just an example for a benchmark tests, I'm mostly worried it sets a precedence on how it should be done.

Reply:
In this case since this represents a Schema A (as it comes from the source), I thought it made more sense to let the field names represent what they are in the original logs, since not always they will even be an ECS to represent the original data (as it comes after transformations, compositions from several fields, etc.). Even though I do not have a strong opinion on this one I think in this case it might not be too troublesome tbh.